pages/common/rnm.md

```diff
@@ -0,0 +1,20 @@
+# rnm
+
+> Batch renaming tool.
+> More information: <https://github.com/neurobin/rnm>.
+
+- Rename a file adding a specific text to the filename:
```

Comment:
How doe  it add it to the filename? Does it append, prepend, or do something else?

Reply:
The '/fn/' is for the full file name including extension ('/f/' is without extension), so in this example it would append.
From the man page:
```
  FILENAME
       1. /fn/ : Full name of the file.

       2. /n/ : File name without extension.

       3. /e/ : File extension.

       4. /rn/ : Replaced Name, generated by replace strings.

       5. /pd/ : Parent directory name of the current file or directory.

       6. /wd/ : Current working directory name.

       7. /nsf/: Name (or name string) from name string file.
```